apps/web/src/app/(authenticated)/insights/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Suspense } from 'react';
+import { Title } from '@mantine/core';
+import { useTranslations } from 'next-intl';
+import LoaderCentered from '@/components/misc/loader-centered';
+
+export default function RootLayout({ children }: { children: React.ReactNode }): React.ReactNode {
```

Comment:
```suggestion
export default function InsightsLayout({ children }: { children: React.ReactNode }): React.ReactNode {
```
not sure if we should name the Layout differently, maybe too much hussle

Reply:
Yeah probably not right now